### PR TITLE
header

### PR DESCRIPTION
**Pull request by DenisMuss**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,27 @@
 <head>
 	<meta charset="UTF-8">
 	<title></title>
+	<link rel="stylesheet" href="style.css">
+	<link href="https://fonts.googleapis.com/css2?family=Preahvihear&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
```
Comment (kkiirill) [inline index.html:8]: What is this? Please explain it

File: index.html [hunk 2]
```diff
@@ -3,10 +3,27 @@
...
+
 </head>
-<body>
-	<h1></h1>
-	<p></p>
+<body class="body">
+
```
Comment (kkiirill) [inline index.html:12]: empty line remove it

File: index.html [hunk 3]
```diff
@@ -3,10 +3,27 @@
...
 	<title></title>
+	<link rel="stylesheet" href="style.css">
+	<link href="https://fonts.googleapis.com/css2?family=Preahvihear&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+
 </head>
-<body>
-	<h1></h1>
-	<p></p>
+<body class="body">
```
Comment (kkiirill) [inline index.html:11]: it's a not good practice use class for body, add tag main and rename class not use class body

File: index.html [hunk 4]
```diff
@@ -3,10 +3,27 @@
...
+	
+	<header class="header">
+		<span class="logo">LOGO</span>
+	 		<nav class="nav">
+				<ul>
```
Comment (kkiirill) [inline index.html:16]: where class for ul? Add class for this

File: index.html [hunk 5]
```diff
@@ -3,10 +3,27 @@
...
+					<li class="active"><a href="#"></a>Home</li>
```
Comment (kkiirill) [inline index.html:17]: where class for li? Add class for this

File: index.html [hunk 6]
```diff
@@ -3,10 +3,27 @@
...
+					<li><a href="#"></a>About us</li>
```
Comment (kkiirill) [inline index.html:18]: Why text not inside tag <a>? Must be <a>About us </a>

File: index.html [hunk 7]
```diff
@@ -3,10 +3,27 @@
...
+					<li><a href="#"></a>Profolio</li>
```
Comment (kkiirill) [inline index.html:19]: the same text must be inside tag <a>

File: index.html [hunk 8]
```diff
@@ -3,10 +3,27 @@
...
+					<li><a href="#"></a>News</li>
+					<li class="contact"><a href="#"></a>Contact us</li>
+				</ul>
+			</nav>
```
Comment (kkiirill) [inline index.html:23]: pls update format your code

File: style.css [hunk 9]
```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;800&display=swap');
+
+.body {
+    background-color: black;
+}
```
Comment (kkiirill) [inline style.css:5]: add empty line between class

File: style.css [hunk 10]
```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;800&display=swap');
+
+.body {
+    background-color: black;
```
Comment (kkiirill) [inline style.css:4]: use variable for this https://developer.mozilla.org/ru/docs/Web/CSS/var

File: style.css [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+}
+.a {
+text-decoration: none;
```
Comment (kkiirill) [inline style.css:7]: it's not good practice name .a? What is A? Pls update class for this

File: style.css [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.img {
```
Comment (kkiirill) [inline style.css:10]: What is img? We doesn't use it no our html page? Remove it. If it not uses

File: style.css [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+max-width: 100%;
+}
+
+
+.header {
+    display: flex;
+}
+
+.nav ul {
```
Comment (kkiirill) [inline style.css:19]: not use ul like tag - add class to ul tag and use class for this

File: style.css [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    display: flex;
+    gap: 88px;
+    list-style: none; /* Убирает маркеры (если это список) */
```
Comment (kkiirill) [inline style.css:25]: remove comment, it's not necessary and for future comments must be on english (not use copy paste pls)

File: style.css [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+    padding: 0; /* Убирает внутренние отступы */
```
Comment (kkiirill) [inline style.css:26]: the same

File: style.css [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+    margin: 0; /* Убирает внешние отступы */
```
Comment (kkiirill) [inline style.css:27]: the same

File: style.css [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+    margin-top: 37px; 
+    margin-left: 689px;
```
Comment (kkiirill) [inline style.css:29]: Why this margin is so big? Use another method like justify-content: space-between

File: style.css [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+    display: flex;
+    gap: 88px;
+    list-style: none; /* Убирает маркеры (если это список) */
+    padding: 0; /* Убирает внутренние отступы */
+    margin: 0; /* Убирает внешние отступы */
+    margin-top: 37px; 
+    margin-left: 689px; 
+}
+
+.nav ul li{
```
Comment (kkiirill) [inline style.css:32]: not good practice use class and after tags, use class for all

File: style.css [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+    color: #fff;
```
Comment (kkiirill) [inline style.css:33]: use var for this color

File: style.css [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+
+.layuot {
```
Comment (kkiirill) [inline style.css:37]: Where we use it on our html page? 
 Use formatting correct

File: style.css [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+max-width: 1300px;
+width: 100%;
+display: flex;
+align-items: center;
+justify-content: center;
+}
+
+.active {
+    border-bottom: 5px solid #DC7000;
```
Comment (kkiirill) [inline style.css:46]: why this bottom is so big? Use var for this color

File: style.css [hunk 22]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.contact {
+    position: relative;
+    display: inline-block;
+    width: 134.51px;
```
Comment (kkiirill) [inline style.css:52]: why it's 134.51?

File: style.css [hunk 23]
```diff
@@ -0,0 +1,70 @@
...
+.active {
+    border-bottom: 5px solid #DC7000;
+}
+
+.contact {
+    position: relative;
+    display: inline-block;
+    width: 134.51px; 
+    height: 33.98px; 
+    background-color: orange;
```
Comment (kkiirill) [inline style.css:54]: use var

File: style.css [hunk 24]
```diff
@@ -0,0 +1,70 @@
...
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    font-size: 29px;
+    display: flex;
+    width: 82px;
+    height: 45px;
+    margin-top: 34px; 
+    margin-left: 36px; 
+    color: #fff;
+}
```
Comment (kkiirill) [inline style.css:70]: add EOF

File: style.css [hunk 25]
```diff
@@ -0,0 +1,70 @@
...
+.logo {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    font-size: 29px;
+    display: flex;
+    width: 82px;
+    height: 45px;
+    margin-top: 34px; 
+    margin-left: 36px; 
+    color: #fff;
```
Comment (kkiirill) [inline style.css:69]: use var

File: index.html (hunk 1 above)
Comment (DenisMuss) [reply index.html:8]: link, these are links. One refers to a style folder by plugging it in, the second refers to Google font - there it patches fonts, the third is a variety of styles

File: index.html (hunk 2 above)
Comment (DenisMuss) [reply index.html:12]: removed

File: index.html (hunk 8 above)
Comment (DenisMuss) [reply index.html:23]: everything is clearly

File: style.css (hunk 11 above)
Comment (DenisMuss) [reply style.css:7]: I removed it until I know how to use it

File: style.css (hunk 12 above)
Comment (DenisMuss) [reply style.css:10]: removed

File: style.css (hunk 13 above)
Comment (DenisMuss) [reply style.css:19]: I removed the nav because it is superfluous there, a big doll from empty tags to get

File: style.css (hunk 15 above)
Comment (DenisMuss) [reply style.css:26]: has done

File: style.css (hunk 16 above)
Comment (DenisMuss) [reply style.css:27]: has done

File: style.css (hunk 22 above)
Comment (DenisMuss) [reply style.css:52]: i fixed

File: style.css (hunk 23 above)
Comment (DenisMuss) [reply style.css:54]: i fixed

**Pull request merged**